sw/device/lib/runtime/meson.build

```diff
@@ -85,6 +85,7 @@ sw_lib_runtime_otbn = declare_dependency(
       sw_lib_dif_otbn,
       sw_lib_mmio,
       sw_lib_runtime_hart,
+      sw_lib_runtime_log,
```

Comment:
Just to check, is this library only for testing? If not we should probably avoid adding this dependency.

Reply:
Yeah, I think given that we are using a different driver for `silicon_creator` code it is probably fine.